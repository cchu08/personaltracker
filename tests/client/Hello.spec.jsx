import React from 'react';
import { shallow } from 'enzyme';
import Hello from '../../client/src/components/Hello';

describe('Hello component', () => {
  const component = shallow(<Hello />);
  
  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('should have an h1 tag', () => {
    expect(component.find('h1').html()).toContain('Hello World.');
  });
})