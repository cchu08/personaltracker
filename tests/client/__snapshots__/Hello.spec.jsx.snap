// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello component renders correctly 1`] = `
<div>
  <h1>
    Hello World.
  </h1>
</div>
`;

exports[`Hello component should render correctly 1`] = `
<div>
  <h1>
    Hello World.
  </h1>
</div>
`;
